astropy/visualization/interval.py

```diff
@@ -152,23 +192,23 @@ class ZScaleInterval(BaseInterval):
     image : array_like
         Input array.
     nsamples : int, optional
-        Number of points in array to sample for determining scaling factors.
-        Default to 1000.
+        Number of points in array to sample for determining scaling
+        factors.  Default to 1000.
```

Comment:
I don't think `Default to ...` is right. Shouldn't it be `DefaultS to ...` or `Default IS ...`